docs/source/properties_computed.rst

```diff
@@ -0,0 +1,113 @@
+Computed Properties
+===================
+
+Helanal follows the procedure of Sugeta and Miyazawa [Sugeta1967]_. Properties
+are computed for a 'window' of four consecutive :math:`C_α` atoms, and this 
+window then slid along the length of the helix in one-residue steps.
+
+For each window consisting of atoms :math:`c_i`, :math:`c_{i+1}`, 
+:math:`c_{i+2}`, :math:`c_{i+3}`, the vectors :math:`\mathbf{B_1}`, 
+:math:`\mathbf{B_2}`, and :math:`\mathbf{B_3}` joining (respectively) atoms 
+:math:`c_i`--:math:`c_{i+1}`, :math:`c_{i+1}`--:math:`c_{i+2}` and 
+:math:`c_{i+2}`--:math:`c_{i+3}` are calculated, along with the vectors 
+:math:`\mathbf{D_1} = \mathbf{C_1} - \mathbf{C_2}` and 
+:math:`\mathbf{D_2} = \mathbf{C_2} - \mathbf{C_3}`. 
+
+.. image:: images/window.svg
+   :width: 60%
+   :align: center
+
+From these, the helix properties below are computed in each simulation frame.
+These properties are available in ``.results`` as arrays, the shape of which
+depends on the number of residues :math:`n_{res}` (and the property being 
+calculated). **Note that each helix must contain at least 9 residues.**
+
+If multiple helices are being analysed, the results are returned as lists (of
+length :math:`n_{helices}`) of arrays of the indicated shape.
+
+All angles are in degrees.
+
+.. list-table::
+   :widths: 40 30 20
+   :header-rows: 1
+
+   * - Description
+     -  
+     - Shape
+   * - ``local_helix_directions``: the normalised vector :math:`\mathbf{D_1}` 
+       (or :math:`\mathbf{D_2}`) for each atom :math:`c_{i+1}` (or 
+       :math:`c_{i+2}`).
+
+       Assuming ~even spacing of the atoms, this vector will bisect the angle 
+       formed by (:math:`c_i,c_{i+1},c_{i+2}`), lie approximately in the plane
+       perpendicular to the helix axis, and point from the projected local 
+       helix centre to the atom :math:`c_{i+1}`.
+     - .. image:: images/helix_directions.svg
+          :width: 100%
+     - :math:`n_{frames}` * :math:`n_{res}`-2 * 3
```

Comment:
Could the \* please be commas -- is that accurate? If so, I find that more intuitive since it's a multi-dimensional shape and not a single flat array. (If the latter we should definitely keep the multiplication sign!)

Reply:
Thanks for doing this!